src/Symfony/Component/Validator/ConstraintViolation.php

```diff
@@ -49,8 +51,11 @@ class ConstraintViolation implements ConstraintViolationInterface
      *                                         caused the violation
      * @param mixed           $cause           The cause of the violation
      */
-    public function __construct(?string $message, ?string $messageTemplate, array $parameters, $root, ?string $propertyPath, $invalidValue, int $plural = null, $code = null, Constraint $constraint = null, $cause = null)
+    public function __construct($message, ?string $messageTemplate, array $parameters, $root, ?string $propertyPath, $invalidValue, int $plural = null, $code = null, Constraint $constraint = null, $cause = null)
     {
+        if (!(null === $message || \is_string($message) || (\is_object($message) && method_exists($message, '__toString')))) {
```

Comment:
`if (!\is_string($message ?? '') && !(\is_object($message) && method_exists($message, '__toString'))) {`

Reply:
It appears that passing a NULL message has been deprecated since I originally sent this PR, and the handling code for that makes both the null === $message and the null coalesce here unnecessary in the current code, as $message cannot be null.

However, the handling code that deprecated passing null as a message  was under the assumption that there would be a 'string' typehint beginning in symfony 5, and if we are going to allow stringable objects, then we cannot typehint a string there.